packages/client/src/views/LoginView.vue

```diff
@@ -1,78 +1,101 @@
 <template>
-  <b-card
-    no-body
-    tag="div"
-    style="max-width: 26rem; margin-top: 11rem"
-    class="mx-auto border-0"
-  >
-    <b-card-img
-      :src="logo"
-      style="max-width: 14rem"
-      alt="United States Digital Response logo"
-      class="mx-auto mb-3"
-      top
-    />
-    <b-card-text class="mt-4 p-3 dropshadow-card">
-      <h1 class="h3 my-4">
-        Log in to Federal Grant Finder
-      </h1>
-      <p class="mb-4">
-        Enter your email to receive a one-time login link.
-      </p>
-      <form @submit="login">
-        <div>
-          <label for="email">Email</label>
-          <input
-            id="email"
-            v-model="email"
-            class="form-control mb-4"
-            name="email"
-            placeholder="Email address"
-            autofocus
+  <main>
```

Comment:
Note to review:  Fixing an accessibility issue - all content should be within a main landmark.